test/intl402/Temporal/TimeZone/non-canonical-antarctica.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.timezone
+description: >
+  TimeZone constructor canonicalises its input.
+features: [Temporal]
+---*/
+
+const testCases = {
+  "Antarctica/South_Pole": "Antarctica/McMurdo",
```

Comment:
ECMA-402 assumes that all links are in `backward`, which is a [known error](https://tc39.es/archives/bugzilla/1892/). For example following the spec by the letter means that `"Etc/GMT"` is canonicalised to `"UTC"` (per [CanonicalizeTimeZoneName](https://tc39.es/ecma402/#sec-canonicalizetimezonename), step 3), whereas `"GMT"` shouldn't be canonicalised, because the `GMT` &rarr; `Etc/GMT` link is in [etcetera](https://github.com/eggert/tz/blob/bf8aa9414b0de5fc9df5f25fa12d07451510e926/etcetera#L26). 

https://github.com/tc39/ecma402/issues/121#issuecomment-323929572 proposes to follow all links in the main time zone files (i.e. excluding `backzone`), but implementers are unlikely to completely ignore `backzone` because of the possible negative cultural/political effects mentioned in https://github.com/tc39/ecma402/issues/272#issuecomment-423928522.

Reply:
If the links for Antarctica/South_Pole in `backward` and `backzone` conflict, then which one should take precedence in your opinion?

(If this is an existing web reality issue then it seems better to me that we only check that the link names _can_ be canonicalized, and not what they are canonicalized to...)

Reply:
I've updated the tests to only check that link names are accepted.